File: backend/src/main/java/com/backend/api/question/dto/response/AIQuestionCreateResponse.java
package com.backend.api.question.dto.response;

import java.util.List;
import java.util.UUID;

public record AIQuestionCreateResponse(
        UUID groudId,
        List<AiQuestionResponse> questions
) {
    public static AIQuestionCreateResponse from(UUID groudId, List<AiQuestionResponse> responses){
        return new AIQuestionCreateResponse(groudId,responses);
    }
}

File: backend/src/main/java/com/backend/api/question/dto/response/AIQuestionCreateResponse.kt
package com.backend.api.question.dto.response

import java.util.*

data class AIQuestionCreateResponse(
    val groudId: UUID?,
    val questions: List<AiQuestionResponse>
) {
    companion object {
        fun from(groudId: UUID?, responses: List<AiQuestionResponse>): AIQuestionCreateResponse {
            return AIQuestionCreateResponse(groudId, responses)
        }
    }
}
